### PR TITLE
AOM-153: Login component does not check if user is logged in on load.

### PR DESCRIPTION
**Pull request by patlub**
## JIRA TICKET NAME:

[AOM-153: Login component does not check if user is logged in on load.](https://issues.openmrs.org/browse/AOM-153)

### SUMMARY:

If a user is already logged in, When they try to access the login page, they should be redirected to the index page. This also avoids multiple login attempts.

### CODE REVIEW

File: app/js/components/manageApps/Login.jsx [hunk 1]
```diff
@@ -9,14 +10,36 @@ class Login extends Component {
...
+  }
+
+  checkLoginStatus(){
+    this.fetchLocation('/v1/session').then((response) => {
+      (response.authenticated) ? hashHistory.push('/') : null;
+    });
+  }
+
+  fetchLocation(url) {
+    const apiHelper = new ApiHelper(null);
```
Comment (segunolalive) [inline app/js/components/manageApps/Login.jsx]: Is the value null being used specifically? Why not call the ApiHelper constructor without arguments instead

**Review by segunolalive — COMMENTED** [review]
Looks good

File: app/js/components/manageApps/Login.jsx (hunk 1 above)
Comment (patlub) [reply app/js/components/manageApps/Login.jsx]: Fixed.

**Review by malmike — COMMENTED** [review]
Looks good to me

**Review by aenkya — COMMENTED** [review]
Looks good to me

**Pull request merged**